backend/internal/models/student_work.go

```diff
@@ -0,0 +1,77 @@
+package models
+
+import "time"
+
+type StudentWork struct {
+	ID                       int        `json:"student_work_id" db:"student_work_id"`
+	ClassroomID              int        `json:"classroom_id" db:"classroom_id"`
+	AssignmentName           *string    `json:"assignment_name" db:"assignment_name"`
+	AssignmentOutlineID      int        `json:"assignment_outline_id" db:"assignment_outline_id"`
+	RepoName                 *string    `json:"repo_name" db:"repo_name"`
+	DueDate                  time.Time  `json:"due_date" db:"due_date"`
+	SubmittedPRNumber        *int       `json:"submitted_pr_number" db:"submitted_pr_number"`
+	ManualFeedbackScore      *int       `json:"manual_feedback_score" db:"manual_feedback_score"`
+	AutoGraderScore          *int       `json:"auto_grader_score" db:"auto_grader_score"`
+	SubmissionTimestamp      *time.Time `json:"submission_timestamp" db:"submission_timestamp"`
+	GradesPublishedTimestamp *time.Time `json:"grades_published_timestamp" db:"grades_published_timestamp"`
+	WorkState                string     `json:"work_state" db:"work_state"`
+	CreatedAt                time.Time  `json:"created_at" db:"created_at"`
+	GitHubUsername           string     `json:"github_username" db:"github_username"`
+	FirstName                string     `json:"first_name" db:"first_name"`
```

Comment:
Having FirstName LastName fields in every StudentWork model is odd to me. The database schema doesn't have these fields, as they are obtained by joining with the user table. 

What does it mean for a "FormattedStudentWork" to have a FirstName and LastName field? Which student's first name is put into there when there is also a list of contributors?